ui/src/components/Sidebar/Sidebar.tsx

```diff
@@ -15,6 +15,10 @@ import {SidebarRow} from './SidebarRow';
 import styles from './styles.module.css';
 
 export const Sidebar: React.FC<{}> = () => {
+  const logoutHandler = () => {
+    window.sessionStorage.removeItem('token');
+    window.location.href = "/login"
```

Comment:
Try using `useNavigate` hook from react-router-dom. That way we can prevent page reload.